File: day01/js/part1.js
//Read Data
var fs  = require("fs");
var array = fs.readFileSync("input.txt").toString().split('\n');
//console.log(array);

for (i = 0; i < array.length; i++) {
	for (j = 0; j < array.length; j++) {
		if(parseInt(array[i])+parseInt(array[j])==2020){
			//throw new Error(parseInt(array[i])*parseInt(array[j]));
			console.log(parseInt(array[i])*parseInt(array[j]))
			break;
		}		
		}
	}

File: day01/r/part1.R
library(dplyr)

input = read.delim("input.txt", header = FALSE) %>% pull()

for (i in input) {
  for (j in input) {
    if (i + j == 2020) {
      print(i)
      print(j)
      print(paste("Final Answer:", i * j))
      break
    }
  }
}